scripts/release/build.ts

```diff
@@ -174,7 +186,7 @@ export const createManifest = (): any => {
 const writeManifest = async () => {
   const dest = new URL('package.json', targetdir);
   const manifest = createManifest();
-  await fs.writeFile(dest, JSON.stringify(manifest));
+  await fs.writeFile(dest, JSON.stringify(manifest, null, 2));
```

Comment:
The single line JSON was really annoying for debugging... and it's convenient to be able to actually read this file when checking on npmjs.org's "Code" tab. Not sure if it was intentionally minified, but even if so, writing with nice whitespace doesn't substantially change the package size.

Reply:
Could you provide a comparison of file size before and after this change?  Some of our consumers have very tight storage limitations they must adhere to, and it is a package used in many other projects, so it is important we keep the package as small as possible.

Reply:
This is purely for the `package.json` file, this change has changed the package size less than the prior two releases which added a few entries to the `data.json` data. The main `data.json` file remains minified.

It is a difference of 190 characters, which is less than the size change from the recent commit adding `WebTransportSendStream`

Reply:
Ah, I misread the code and assumed this was altering `data.json`, my mistake.  `package.json` is fine to unminify.  👍